server/src/lib/data/update-habit-entry.ts

```diff
@@ -0,0 +1,24 @@
+import { sqlConnection } from "@/db/init";
+import type { RequestHandler } from "express";
+import type { HabitEntry, HabitEntryUpdateInput } from "types/data/habit.types";
+import type { WithSQL } from "types/sql.types";
+
+// TODO: rename this file to put-habit-entry
```

Comment:
Handle this TODO? I'm not sure if that is what I want to do; check other files and see what convention I picked.

Reply:
Thought about this. Turn this into a new issue.